docs/main_page.md

```diff
@@ -0,0 +1,49 @@
+# CyberWay.CDT
+
+## Contract Development Toolkit based on EOSIO.CDT
+
+### Version : 1.6.3
+
+CyberWay.CDT is a toolchain for WebAssembly (WASM) and set of tools to facilitate contract writing for the CyberWay platform.  In addition to being a general purpose WebAssembly toolchain, [CyberWay](https://github.com/cyberway/cyberway) specific optimizations are available to support building CyberWay smart contracts.  This toolchain is built around [Clang 7](https://github.com/eosio/llvm), which means that CyberWay.CDT has the most currently available optimizations and analyses from LLVM, but as the WASM target is still considered experimental, some optimizations are not available or incomplete.
+
```

Comment:
I think it's need to say some words about EOS as base for cyberway-toolchain. CDT provides all eosio tools and adds specific for cyberway (`cyberway-abidiff` and `cyberway-cpp`)